src/main.rs

```diff
@@ -48,14 +48,17 @@ fn init(mut opt: cli::Opt) -> Result<()> {
 
     config.save(opt.find_amber_yaml_or_default())?;
 
-    eprintln!("Your secret key is: {}", secret_key);
-    eprintln!(
-        "Please save this key immediately! If you lose it, you will lose access to your secrets."
-    );
-    eprintln!("Recommendation: keep it in a password manager");
-    eprintln!("If you're using this for CI, please update your CI configuration with a secret environment variable");
-    println!("export {}={}", config::SECRET_KEY_ENV, secret_key);
-
+    if opt.only_secret_key {
+        println!("{}", secret_key);
```

Comment:
If I understand the use case correctly: would it make more sense to use `print!` instead to avoid an extra newline?

Reply:
exactly, thanks
fix it